interactions/api/gateway/client.py

```diff
@@ -112,7 +112,7 @@ def __init__(
         except RuntimeError:
             self._loop = new_event_loop()
         self._dispatch: Listener = Listener()
-        self._http: HTTPClient = HTTPClient(token)
```

Comment:
I don't see why changing the http attribute fixes the unclosed client session message tbh

From my testing it pointed from the weird global-esque session definition, http variable was just fine?

Reply:
the attrib indeed causes a unclosed client session message, the global session definition caused smth other that was unclosed

Reply:
(it only happens when you don't start the bot but yeah